dipy/workflows/tests/test_reconst_csa_csd.py

```diff
@@ -132,11 +132,6 @@ def reconst_flow_core(flow):
             # test parallel implementation
             reconst_flow = flow()
             reconst_flow._force_overwrite = True
-            reconst_flow.run(data_path, bval_path, bvec_path, mask_path,
-                             out_dir=out_dir,
-                             parallel=True, num_processes=None)
-            reconst_flow = flow()
-            reconst_flow._force_overwrite = True
```

Comment:
What's the issue with this test?

Reply:
1. Not really a test.
2. avoid using all processors to freeze the machine. since there is already one with `num_processes=2`, no need of this one

Reply:
Makes sense! Off we go!